src/main/java/seedu/address/model/person/Address.java

```diff
@@ -9,13 +9,15 @@
  */
 public class Address {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS =
+        "Addresses can take any values, and it should not be blank\n"
+        + "Addresses cannot be longer than 150 characters";
 
     /*
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[^\\s].*";
+    public static final String VALIDATION_REGEX = "[^\\s].{0,150}";
 
```

Comment:
For the add command, we made it such that adding an address is mandatory. Setting the range from 0 might not be the best idea as it may lead to buggy behavior when adding contacts with empty addresses

Reply:
Ok, i have corrected this